test/Microsoft.ML.AutoML.Tests/AutoMLFailureTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using Microsoft.ML.Data;
+using Microsoft.ML.TestFramework;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.AutoML.Test
+{
+
+    public class AutoMLFailureTests : BaseTestClass
+    {
+        public AutoMLFailureTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        public class ModelInput
+        {
+            [ColumnName("Label"), LoadColumn(0)]
+            public int Label { get; set; }
+
+
+            [ColumnName("ProblematicColumn"), LoadColumn(1)]
+            public string ProblematicColumn { get; set; }
+
+        }
+
+        [Fact]
+        public void CrossValidationOverflowTest()
+        {
+            // This test is introduced for https://github.com/dotnet/machinelearning/issues/5211
+            // that provides users an informational exception message
+            MLContext mlContext = new MLContext(1);
+
+            IDataView trainingDataView = mlContext.Data.LoadFromTextFile<ModelInput>(
+                                            path: GetDataPath("cross_validation_overflow_dataset.txt"),
+                                            hasHeader: true,
```

Comment:
Does this need to be stored as a file? Is it not possible to generate random values in the column and reproduce the same issue?

Reply:
It might be possible, let me try it